### PR TITLE
Add help job

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet bot test outerloop portable windows release pipeline

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@stephentoub @danmosemsft Can you take a look?  Getting this stuff in will allow for us to run win10 and nano by default again.

**Comment by mmitche** [comment]
@dotnet-bot test portable linux debug pipeline
@dotnet-bot test portable linux release pipeline
@dotnet-bot test portable windows debug pipeline
@dotnet-bot test portable windows release pipeline

@dotnet-bot test outerloop portable linux debug pipeline
@dotnet-bot test outerloop portable linux release pipeline
@dotnet-bot test outerloop portable windows debug pipeline
@dotnet-bot test outerloop portable windows release pipeline

File: buildpipeline/pipelinejobs.groovy [hunk 1]
```diff
@@ -28,10 +33,19 @@ def windowsPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'b
 ['netcoreapp'].each { targetGroup ->
 	['Debug', 'Release'].each { configurationGroup ->
 		['Windows x64'].each { osName ->
-            def parameters = ['Config':configurationGroup, 'OuterLoop':false]
-            windowsPipeline.triggerPipelineOnGithubPRComment("${osName} ${configurationGroup} Build", "(?i).*test\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*", parameters)
+            // One for just innerloop
+            windowsPipeline.triggerPipelineOnGithubPRComment("Portable ${osName} ${configurationGroup} Build", "(?i).*test\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*",
+                ['Config':configurationGroup, 'OuterLoop':false])
+            // Add one for outerloop
+            windowsPipeline.triggerPipelineOnGithubPRComment("Portable Outerloop ${osName} ${configurationGroup} Build", "(?i).*test\\W+outerloop\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*",
```
Comment (stephentoub) [inline buildpipeline/pipelinejobs.groovy]: Is there room anywhere to include a description of where things will actually run?  e.g. to tell the caller than this one leg will result in runs on Win7, Win8, Win10, and Nano?

Comment (mmitche) [inline buildpipeline/pipelinejobs.groovy]: Added comments to this. There isn't a good way to have this show up in GitHub.

**Pull request merged**